Core/src/org/sleuthkit/autopsy/modules/hashdatabase/HashDbIngestModule.java

```diff
@@ -90,11 +88,23 @@ private static synchronized IngestJobTotals getTotalsForIngestJobs(long ingestJo
         return totals;
     }
 
-    HashDbIngestModule(HashLookupModuleSettings settings) throws NoCurrentCaseException {
+    /**
+     * Create a HashDbIngestModule object that will mark files based on a
+     * supplied list of hash values. The supplied HashLookupModuleSettings
+     * object is used to configure the module.
+     *
+     * @param settings The module settings.
+     */
+    HashDbIngestModule(HashLookupModuleSettings settings) {
         this.settings = settings;
-        skCase = Case.getCurrentCaseThrows().getSleuthkitCase();
     }
 
+    @Messages({
+        "HashDbIngestModule.noKnownBadHashDbSetMsg=No notable hash set.",
+        "HashDbIngestModule.knownBadFileSearchWillNotExecuteWarn=Notable file search will not be executed.",
+        "HashDbIngestModule.noKnownHashDbSetMsg=No known hash set.",
+        "HashDbIngestModule.knownFileSearchWillNotExecuteWarn=Known file search will not be executed."
+    })
```

Comment:
I don't think we should be converting existing Bundle.properties messages to annotation form. If we do this, we lose the work that has already been done to translate the existing messages into Japanese.

Reply:
What about exception messages that have been translated? I made two of those literal because we usually don't want to keep those as Bundle messages.

Reply:
Please see revised general guidance in email.

Regarding the exception messages, there is a small subset of messages that get displayed to user. If the messages in question go into the log only, then the localization is indeed a mistake and you should proceed.